app/assets/javascripts/discourse/app/widgets/search-menu-results.js

```diff
@@ -73,6 +75,10 @@ export function addQuickSearchRandomTip(tip) {
   }
 }
 
+export function removeDefaultQuickSearchRandomTips() {
```

Comment:
Should this be available via the `pluginApi`?

Reply:
I guess so.. I don't know always b/c I don't think _every_ exported function needs to be in the plugin API, but I can certainly add it there